src/main/java/com/mraof/minestuck/tileentity/redstone/AreaEffectTileEntity.java

```diff
@@ -37,7 +40,7 @@ public void tick()
 		if(level == null || !level.isAreaLoaded(getBlockPos(), 0))
 			return;
 		
-		if(level.getGameTime() % 80 == 0 && !level.isClientSide && level.getBlockState(getBlockPos()).getBlock() instanceof AreaEffectBlock && getBlockState().getValue(AreaEffectBlock.POWERED))
+		if(level.getGameTime() % 80 == 0 && !level.isClientSide && level.getBlockState(getBlockPos()).getBlock() instanceof AreaEffectBlock && getBlockState().getValue(AreaEffectBlock.POWERED) && !getBlockState().getValue(AreaEffectBlock.SHUT_DOWN))
```

Comment:
I think that you should be able to assume that `level.getBlockState(getBlockPos()).getBlock() instanceof AreaEffectBlock` cannot be false